server/package.json

```diff
@@ -45,9 +46,13 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
+    "body-parser": "^1.18.2",
     "express": "^4.16.2",
+    "helmet": "^3.12.0",
     "immutable": "^3.8.1",
+    "js-string-escape": "^1.0.1",
     "redux": "^3.7.2",
-    "serverless-http": "^1.5.2"
+    "serverless-http": "^1.5.2",
+    "uuid": "^3.2.1"
```

Comment:
so is this like... a regex?

Reply:
Haha no, this means it can update minor versions. https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/